appengine/flexible/AppEngineFlex.sln

```diff
@@ -8,8 +8,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Analytics", "Analytics\Anal
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "RedisCache", "RedisCache\RedisCache.csproj", "{6E430843-E06E-47C5-ADC2-475C8EDE81DB}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SendGrid", "SendGrid\SendGrid.csproj", "{0C33CDB9-DC3C-4832-AA81-83FE5CAFEF57}"

```

Comment:
You also need to remove the configuration associated to this project, basically entries that reference 0C33CDB9-DC3C-4832-AA81-83FE5CAFEF57. I gather you are not using Visual Sutdion?

Reply:
Nope - I guess that would explain it.

If you can make the changes needed, that'd be much appreciated! 🙂